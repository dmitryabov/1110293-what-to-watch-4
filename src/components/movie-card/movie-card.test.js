import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movie-card.jsx";


const movie =
  {
    id: 1,
    src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    name: `Fantastic Beasts`
  };

it(`Render MovieCard`, () => {

  const tree = renderer
  .create(<MovieCard
    movies={movie}
    onMovieCardClick={() => {}}
    onMouseEnterCard={() => {}}
    onMouseLeaveCard={() => {}}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
