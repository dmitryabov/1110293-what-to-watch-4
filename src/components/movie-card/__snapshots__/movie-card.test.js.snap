// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  id={6}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <img
      alt="What We Do in the Shadows"
      height="175"
      src="img/what-we-do-in-the-shadows.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      What We Do in the Shadows
    </a>
  </h3>
</article>
`;
